packages/wrangler/src/dev.tsx

```diff
@@ -866,10 +866,7 @@ export async function startDev(args: StartDevOptions) {
 							assetsWatcher = watch(assetsOptions.directory, {
 								persistent: true,
 								ignoreInitial: true,
-							}).on("all", async (eventName, changedPath) => {
-								const message = getAssetChangeMessage(eventName, changedPath);
-
-								logger.log(`🌀 ${message}...`);
```

Comment:
I think we should log the message inside the debounce

Reply:
Log something like "x files in assets directory changed"?